rdkit/Chem/UnitTestInchi.py

```diff
@@ -167,9 +165,9 @@ def test0InchiWritePubChem(self):
 
       fmt = "\n{0}InChI write Summary: {1} identical, {2} suffix variance, {3} reasonable{4}"
       print(fmt.format(COLOR_GREEN, same, diff, reasonable, COLOR_RESET))
-      self.assertEqual(same, 1160)
+      self.assertEqual(same, 1162)
```

Comment:
I love it when this number gets larger. :+1:

Reply:
Me too!